PayPal/src/test/java/com/braintreepayments/api/paypal/MockPayPalInternalClientBuilder.java

```diff
@@ -9,6 +9,10 @@
 import org.mockito.invocation.InvocationOnMock;
 import org.mockito.stubbing.Answer;
 
+/**
+ * This outputs a Mockito mock of PayPalInternalClient. For newer kotlin tests, please use MockkPayPalInternalClientBuilder that outputs a mockk mock.
+ */
```

Comment:
```suggestion
 * @deprecated Use {@link com.braintreepayments.api.testutils.MockkBraintreeClientBuilder} instead.
 */
```

Might be helpful to have a link to the class's replacement.

Reply:
`MockkBraintreeClientBuilder` is not a replacement for MockBraintreeClientBuilder, though. We'd want to use `MockkBraintreeClientBuilder` in case the implementation requires mockk object and not a Mockito object.

Reply:
That's fair. Maybe we can just add the link annotation to where we call out `MockkPayPalInternalClientBuilder` to make navigating files easier.

Reply:
@tdchow your thoughts?

Reply:
I agree, we don't want to call it a direct replacement. All I'm calling out is possibly adding the @link annotation to make files easier to navigate.

Reply:
> @tdchow your thoughts?

I should refresh more often :/